src/crimson/mgr/client.cc

```diff
@@ -102,27 +103,19 @@ seastar::future<> Client::handle_mgr_conf(crimson::net::Connection* conn,
   logger().info("{} {}", __func__, *m);
   tick_period = std::chrono::seconds{m->stats_period};
   if (tick_period.count() && !tick_timer.armed() ) {
```

Comment:
- It seems the new `tick_period` won't be able to apply to the timer if it is already armed.
- `tick_period` doesn't need to be stored as class member after this change.

Reply:
> * `tick_period` doesn't need to be stored as class member after this change.

why?

Reply:
It can be stored in stack memory if only `handle_mgr_conf()` is using it, after this change.

Reply:
@cyx1231st updated and repushed.